src/veh_interact.cpp

```diff
@@ -572,13 +589,13 @@ void veh_interact::do_install()
     }
     mvwprintz(w_mode, 0, 1, c_ltgray, _("Choose new part to install here:"));
     wrefresh (w_mode);
-    
+
```

Comment:
If you look at the diff above, it looks like it's this PR that's attempting to delete the trailing space. If you deleted it already, I wouldn't have had a problem.